packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -183,7 +183,6 @@ function PreviewView() {
 
   function onInspectorItemSelected(item: InspectDataStackItem) {
     openFileAt(item.source.fileName, item.source.line0Based, item.source.column0Based);
-    setIsInspecting(false);
```

Comment:
Right, onInspectorItemSelected is also called in InspectDataMenu on element selection, however in that case isInspecting is already set to false, while  only !!inspectFrame is changed to true. 

If onInspectorItemSelected were to set isInspecting to false at first onMouseDown event after right-click (which calls sendInspect and uses onInspectorItemSelected function) in Preview, it would called resetInspector too early. We want to set isInspecting to false not until onMouseUp, so in that way we keep element highlighted on first right click (first onMouseUp event) and reset it on second (the next onMouseUp event).